prisma/migrations/20240717125124_profile_updated/migration.sql

```diff
@@ -0,0 +1,14 @@
+/*
+  Warnings:
+
+  - You are about to drop the column `githubUrl` on the `Profile` table. All the data in the column will be lost.
```

Comment:
We can probably remove this from the migration, as we aren't actually dropping the column